src/core/Checkbox/Checkbox.tsx

```diff
@@ -94,67 +99,45 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
       }
     });
 
```

Comment:
Can you check why spacing differs?
![image](https://user-images.githubusercontent.com/36186912/154936846-00252719-6d34-45cf-93ac-059abc8ea653.png)

Reply:
It's because unlabeled checkbox (bare `<input>`) has a height of 16px whereas labeled one has 22px
![image](https://user-images.githubusercontent.com/9084735/155201184-3fc18930-c54e-419e-867c-39c700338cf6.png)

We'll need to fix this somehow 🤔

Reply:
Or maybe it's ok. InputGroup should have labeled checkboxes anyway. This story example feels unrealistic

Reply:
Added "Select all" label to that checkbox.